src/pkg/cli/tail.go

```diff
@@ -247,7 +247,7 @@ func tail(ctx context.Context, client client.Client, params TailOptions) error {
 								modeStr += ". I like the way you work it, no verbosity."
 							}
 							term.Info("Verbose mode", modeStr)
-							go client.Track("Verbose Toggled", P{"verbose", verbose}, P{"toggleCount", toggleCount})
+							// go client.Track("Verbose Toggled", P{"verbose", verbose}, P{"toggleCount", toggleCount}) // FIXME: Do we need to pass client around to track?
```

Comment:
restore?

Reply:
I'll have to tackle it in the next PR, if I use command.Track it becomes a circular depencency, if I pass the client all the way down from the commands, it touches too many function signature that would be restored once I have the track package restructuring done. I've added a fixme for future reference.